app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ProfilesController < ApplicationController
```

Comment:
It's worth noting that in that same section of the Rails Guide, it mentions that singular resources map to plural controllers, so

`resource :profile` will generate a route that looks like `/profile` but it will still expect to see all of this coming from `ProfilesController`